docker/bundle.Dockerfile

```diff
@@ -1,14 +1,3 @@
-# Copyright (c) 2023-2024 Red Hat, Inc.
```

Comment:
License header was after a regeneration. But I think it should be fine since we have a [LICENSE file](https://github.com/janus-idp/operator/blob/main/LICENSE) in the repo.

Reply:
@nickboldt is this fine?

Reply:
well, the header is better. :D see my additional commit on this PR to make it reappear. 

Note also that `operator-sdk-v1.33.0` is out now, and throws some new warnings when run.

Reply:
Sounds good - thanks! We can look into fixing those warnings in a subsequent issue/PR.